### ISSUE TITLE
The email for travis notifications is mis-configured

### ISSUE THREAD

**Issue by maysamyabandeh** [YugaByte/yugabyte-db#40]
https://github.com/YugaByte/yugabyte-db/blob/master/.travis.yml#L41
```
notifications:
    email:
      - leveldb@fb.com
    webhooks:
```

**Comment by rkarthik007** [comment]
Hi @maysamyabandeh - so sorry about that, my fault. Will fix it right away. Was exploring how to integrate with Travis but didnt realize I had not changed the starting template.

**Comment by mbautin** [comment]
Fixed in 491c1ac616ac4319ef20a9dd347c9e241f05bbc9.
